main.py

```diff
@@ -148,5 +148,75 @@ async def keen(context):
     await context.send(result)
 
 
-bot.run(TOKEN)
+@bot.command(name='daily', help='Will give the daily leaderboard for specified day')
+async def daily(context, day):
```

Comment:
Only problem will be at 12-5 in the morning.  Will assume you are on the next day when we are still on the previous day.  At the moment line 175 catches all wrong inputs.  (I have added a check in case you haven't entered any day)